taxi/forms.py

```diff
@@ -1,52 +1,64 @@
 from django import forms
 from django.contrib.auth import get_user_model
 from django.contrib.auth.forms import UserCreationForm
-from django.core.exceptions import ValidationError
+from django.core.validators import RegexValidator
 
-from taxi.models import Car, Driver
+from taxi.models import Driver, Car
 
 
-class CarForm(forms.ModelForm):
-    drivers = forms.ModelMultipleChoiceField(
-        queryset=get_user_model().objects.all(),
-        widget=forms.CheckboxSelectMultiple,
-    )
-
-    class Meta:
-        model = Car
-        fields = "__all__"
+DRIVER_LICENSE_VALIDATOR = RegexValidator(
+    regex=r"^[A-Z]{3}[\d]{5}$",
+    message="Enter a valid value license number:"
+            " 1.Consist only of 8 characters"
+            " 2.First 3 characters are uppercase letters"
+            " 3.Last 5 characters are digits", )
 
 
 class DriverCreationForm(UserCreationForm):
+    license_number = forms.CharField(
+        required=True,
+        validators=[DRIVER_LICENSE_VALIDATOR, ]
+    )
+
     class Meta(UserCreationForm.Meta):
         model = Driver
         fields = UserCreationForm.Meta.fields + (
-            "license_number",
             "first_name",
             "last_name",
+            "license_number",
         )
 
-    def clean_license_number(self):  # this logic is optional, but possible
-        return validate_license_number(self.cleaned_data["license_number"])
-
 
 class DriverLicenseUpdateForm(forms.ModelForm):
+    license_number = forms.CharField(
+        required=True,
+        validators=[DRIVER_LICENSE_VALIDATOR, ]
+    )
+
     class Meta:
         model = Driver
-        fields = ["license_number"]
+        fields = ("license_number",)
+
+
+class CarForm(forms.ModelForm):
+    drivers = forms.ModelMultipleChoiceField(
+        queryset=get_user_model().objects.all(),
```

Comment:
Ensure that `get_user_model()` returns the `Driver` model or a compatible model. If `Driver` is not the user model, consider using `Driver.objects.all()` instead.